app/src/main/res/values/strings.xml

```diff
@@ -320,6 +320,7 @@
     <string name="two_factors_otp_error">Two factors OTP is required</string>
     <string name="no_issue">No issue</string>
     <string name="copy">Copy</string>
+    <string name="copy_repo_url">Copy repo URL</string>
```

Comment:
Well I did not know if this was being used else where so I did not want to change the it.

Whats final? Take it out?

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_